opamp/testdata/package_statuses_good.yaml

```diff
@@ -0,0 +1,25 @@
+packages:
```

Comment:
Would it make more sense to put these under a `observiq` package (e.g. `opmap/observiq/testdata` instead of `opamp/testdata`)? Since it's only being used in that package.

Reply:
i'm a little on the fence for this one. i'm ok with having test files in one nice spot unless that spot gets unwieldy and needs to be split up. if you feel strongly about it though, let me know

Reply:
I don't feel super strongly about it, it's just where I'd put it. Should be fine in that spot.